internal/vds/vds.cpp

```diff
@@ -520,6 +520,26 @@ struct response fetch_horizon_metadata(
     return to_response(meta);
 }
 
+struct Context {
```

Comment:
Mostly because we start exposing C++ classes (DataHandle) in this PR and they inherit their name from C++ where we use uppercase for classes/structs. I didn't bother renaming `response`, as it might be removed completely in favor of buffer allocation in go